Sources/AudioKit/Nodes/Playback/AudioPlayer/AudioPlayer.swift

```diff
@@ -26,14 +26,8 @@ public class AudioPlayer: Node {
         set { playerNode.volume = newValue }
     }
 
-    /// Whether or not the playing is playing
-    public internal(set) var isPlaying: Bool = false
-
-    /// Whether or not the playing is paused
-    public internal(set) var isPaused: Bool = false
-
-    /// Will be true if there is an existing schedule event
-    public var isScheduled: Bool { scheduleTime != nil }
+    /// Status of the player node (playing, paused, stopped, or scheduling
+    public internal(set) var status = NodeStatus.Playback.stopped
```

Comment:
Initialization for the node status. This is what I was thinking for each type of node. Then more enums could be added to the whole status enum as they are used with other nodes.